metal/multitask/mt_label_model.py

```diff
@@ -12,39 +13,54 @@ def __init__(self, K=None, task_graph=None, **kwargs):
             K: A t-length list of task cardinalities (overrided by task_graph
                 if task_graph is not None)
             task_graph: TaskGraph: A TaskGraph which defines a feasible set of
-                task label vectors; overrides K
+                task label vectors; overrides K if provided
         """
         config = recursive_merge_dicts(lm_default_config, kwargs)
         MTClassifier.__init__(self, K, config)
 
-        # WARNING: Currently only storing 1 k for all tasks in self.k and
-        # ignoring self.K!
-        print("WARNING: Currently assuming all tasks have same cardinality!")
+        if task_graph is None:
+           task_graph = TaskGraph(K)
         self.task_graph = task_graph
-        if self.task_graph is None:
-            self.k = K  # TODO: this should not typecheck! scalar == list
-        else:
-            self.k = len(self.task_graph)
+
+        # Note: While K is a list of the cardinalities of the tasks, k is the 
+        # cardinality of the feasible set. These are always the same for a 
+        # single-task model, but rarely for a multi-task model.
+        self.K = self.task_graph.K  
+        self.k = self.task_graph.k
 
     def _set_constants(self, L):
         self.n, self.m = L[0].shape
         self.t = len(L)
 
-    def _create_L_ind(self, L, km, offset):
-        L_ind = np.ones((self.n, self.m * km))
+    def _create_L_ind(self, L):
+        """Convert T label matrices with labels in 0...K_t to a one-hot format
```

Comment:
As long as expanding into a full docstring, I'd add a bit more here- eg
```
Here we can view e.g. the $(i,j)$ entries of the $T$ label matrices as a _label vector_ emitted by LF j for data point i.
```
Or something like this?